docs/spfx/compatibility.md

```diff
@@ -0,0 +1,60 @@
+---
+title: SharePoint Framework development tools and libraries compatibility
+description: Use client-side JavaScript libraries to build your solutions and develop client-side web parts.
+ms.date: 03/20/2020
+ms.prod: sharepoint
+localization_priority: Priority
+---
+
+
+# SharePoint Framework development tools and libraries
+
+As the SharePoint Framework evolves, so do the various development tools and libraries that it uses.
+
+## SharePoint Framework version compatibility
+
+Because SharePoint Online and the on-premises versions of SharePoint Server have different release cycles for new capabilities, they also have support different Sharepoint Framework capabilities.
+
+SharePoint Online always uses the latest version of the SharePoint Framework, but SharePoint 2016 and SharePoint 2019 only support the versions that match the server-side dependencies of the deployed packages.
+
+|SharePoint version|Supported SPFx version|Supported features
+|---|---|---|
+|SharePoint Online|All versions|All features
+|SharePoint Server 2019|v1.4.1 or lower|SPFx client-side web parts in classic and modern pages, and extensions in modern pages.
+|SharePoint 2016 Feature Pack 2|v1.1|SPFx client-side web parts hosted in classic SharePoint pages.
+
+For more information about SharePoint Framework development with SharePoint 2016 Feature Pack 2 and SharePoint 2019, refer to the following:
+
+* [SharePoint Framework development with SharePoint 2016 Feature Pack 2](sharepoint-2016-support.md)
+* [SharePoint Framework development with SharePoint Server 2019](sharepoint-2019-support.md)
+
+## SPFx development environment compatibility
+
+As each new version of the SharePoint Framework is released, support for newer versions libraries is constantly added to ensure that the toolset remains up-to-date.
+
+The following table lists each version of SharePoint Framework along with each library versions are compatible:
+
+|SPFx|Node.js|NPM|TypeScript|React
+|---|---|---|---|---|---|
```

Comment:
Added the additional dash which is causing incorrect table alignment

Reply:
Thanks for pointing it out. fixed the table as per your suggestion